### DISCUSSION TITLE
Ethereum Guild 2024-01 (December 11, 2024)

### DISCUSSION THREAD

**Discussion by philknows** [ChainSafe/lodestar#7277] [Ethereum Guild]
When: [December 11, 2024, 15:00-16:00 UTC](https://savvytime.com/converter/utc-to-pst-est-cet-ist-ict-aedt/dec-11-2024/3pm)
Zoom: https://chainsafe-io.zoom.us/j/81714567108?pwd=7zcxxLVMJOcSUaRc8nMb9rWLx1o5al.1

## Topics for discussion:
- [EIP-7742: Uncouple blob count between CL and EL](https://eips.ethereum.org/EIPS/eip-7742)
  - Reading: https://hackmd.io/@ralexstokes/rJhC_vkSR#Uncertainty-around-deployment-timeline-and-cross-layer-concerns

- [EIP-7691: Blob throughput increase](https://eips.ethereum.org/EIPS/eip-7691)
  - Reading: https://ethresear.ch/t/bandwidth-availability-in-ethereum-regional-differences-and-network-impacts/21138
  - Reading: https://ethresear.ch/t/block-arrivals-home-stakers-bumping-the-blob-count/21096
  - Reading: https://ethresear.ch/t/on-blob-markets-base-fee-adjustments-and-optimizations/21024

- [EIP-7594: PeerDAS - Peer Data Availability Sampling](https://eips.ethereum.org/EIPS/eip-7594)
  - [Consensus Spec](https://github.com/ethereum/consensus-specs/tree/dev/specs/_features/eip7594)
  - Reading: https://hackmd.io/@manunalepa/peerDAS
  - Video: https://youtu.be/toR2UKzE_zA?si=RoeYG7Lb94g0kOev

Please add to this thread or edit this post to include additional material to build context before attending the guild meeting. Additional topics, questions, or discussion points are welcome!

**Comment by matthewkeil** [comment]
@nalepae would like to join!  Pinging him so he can see our agenda and notes

**Comment by matthewkeil** [comment]
@zilm13 would like to join!  Pinging him so he can see our agenda and notes

**Comment by philknows** [comment]
Closing for the first Ethereum Guild of 2025: https://github.com/ChainSafe/lodestar/discussions/7332
Recording available upon request.
